2020_Fall/readme.md

```diff
@@ -43,7 +43,7 @@ Make the Pull Request from your own GitHub account. Make sure you're changing th
 
 - [ ] Esteves, Thais
 
-- [ ] Gontijo Bisinoto, Luisa
```

Comment:
Looks like you've missed adding the GitHub username and emoji. Just make another edit on the file, on this branch itself.